npm2deb/utils.py

```diff
@@ -136,6 +136,10 @@ def create_dir(dir):
     except OSError as oserror:
         raise OSError("Error: directory %s already exists." % oserror.filename)
 
+def parse_name(name):
+    parts = name.partition('@')
```

Comment:
Are you sure that all the modules have this format ?

Reply:
we are parsing user input here.
if `@version` is not provided, npm defaults to latest version